kolibri/plugins/coach/assets/src/views/lessons/LessonResourceSelectionPage/index.vue

```diff
@@ -0,0 +1,244 @@
+        </router-link>
+      </div>
+    </ui-toolbar>
+
+    <div class="immersive-content">
+      <form class="selection-form" @submit.prevent="saveResources">
+        <h1 class="selection-header">{{ $tr('addResourcesHeader') }}</h1>
+
+        <search-box />
+
+        <k-breadcrumbs
+          :items="selectionCrumbs"
+          :showAllCrumbs="true"
+        />
+
+        <ul class="content-list">
+          <li
+            class="content-list-item"
+            :key="content.id"
+            v-for="content in contentList"
+          >
+            <k-checkbox
+              class="content-checkbox"
+              :label="content.title"
+              v-if="!contentIsDirectoryKind(content)"
+              :showLabel="false"
+              :checked="isSelected(content.id)"
+              @change="toggleSelected($event, content.id)"
+            />
+            <content-card
+              class="content-card"
+              :title="content.title"
+              :thumbnail="content.thumbnail"
+              :description="content.description"
+              :kind="content.kind"
+              :link="contentLink(content)"
+            />
+          </li>
+        </ul>
+
+        <div class="information">
+          <p> {{ $tr('totalResourcesSelected', { total: workingResources.length }) }} </p>
+          <k-button
+            type="submit"
+            :primary="true"
+            :text="$tr('save')"
+          />
+        </div>
+      </form>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import uiToolbar from 'keen-ui/src/UiToolbar';
+  import contentCard from './content-card';
+  import kBreadcrumbs from 'kolibri.coreVue.components.kBreadcrumbs';
+  import kButton from 'kolibri.coreVue.components.kButton';
+  import kCheckbox from 'kolibri.coreVue.components.kCheckbox';
+  import { saveLessonResources } from '../../../state/actions/lessons';
+  import { createSnackbar } from 'kolibri.coreVue.vuex.actions';
+  import { LessonsPageNames } from '../../../lessonsConstants';
+  import searchBox from '../../../../../../learn/assets/src/views/search-box/';
+  import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+  import { lessonSummaryLink, selectionRootLink, topicListingLink } from '../lessonsRouterUtils';
+
+  export default {
+    name: 'lessonResourceSelectionPage',
+    components: {
+      uiToolbar,
+      kBreadcrumbs,
+      searchBox,
+      contentCard,
+      kButton,
+      kCheckbox,
+    },
+    computed: {
+      lessonPage() {
+        return lessonSummaryLink(this.routerParams);
+      },
+      routerParams() {
+        return { classId: this.classId, lessonId: this.lessonId };
+      },
+      selectionCrumbs() {
+        return [
+          // The "Channels" breadcrumb
+          { text: this.$tr('channelBreadcrumbLabel'), link: selectionRootLink(this.routerParams) },
+          // Ancestors breadcrumbs
+          // NOTE: The current topic is injected into `ancestors` in the showPage action
```

Comment:
maybe we should just include breadcrumbs instead of ancestors at the action level? Only argument for including ancestors I can think of atm is for the channel name on each content card... but we could do that another way